bats/fb-test-foreman.bats

```diff
@@ -10,7 +10,7 @@ set -o pipefail
 }
 
 @test "check web app is up" {
-  curl -sk "https://localhost$URL_PREFIX/users/login" | grep login-form
+  curl -sk "https://localhost$URL_PREFIX/users/login" | grep login-page
```

Comment:
This needs to work with the old page as well since this is also used in the stable branches.